examples/python-kubernetes/Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+cdktf = "~=0.0.12"
```

Comment:
Please use the references [as here](https://github.com/hashicorp/terraform-cdk/blob/b8aeded4cb58a7f367195d5ed8a4632588f43eca/examples/python-aws/Pipfile)

```
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[requires]
python_version = "3.7"

[packages]
cdktf = {path = "./../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Reply:
Made the change 🍰